etc/scripts/management.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+sleep 20
```

Comment:
What's that for? To wait for `init` to finish? This is going to be very unstable, because startup times will vary in different systems. You need a way to reliably detect that everything is up.